### ISSUE TITLE
There's a mismatch between the GitHub `module.json` and the Foundry module entry

### ISSUE THREAD

**Issue by DC23** [DC23/basicfantasyrpg-corerules-en#23]
v0.1.3 `module.json` specifies Foundry minimum 9, verified 10 while the Foundry module listing gives v11 verified:
![image](https://github.com/user-attachments/assets/cfabc180-c993-43f6-b2ba-ac5af052618f)

I know the module works on Foundry v11, so I should look into the inconsistency with the manifest once the ownership of the module listing on Foundry is transferred.

**Comment by Stew-rt** [comment]
I've updated the Package URL and the URLs for the release versions in the Foundry Module, whilst we wait for the transfer to finish.

**Comment by DC23** [comment]
I've decided to close this as "wontfix". The inconsistency is still there, but Foundry v11 doesn't care. They have only started applying more stringent error checking on the manifest in v12. I'm planning to leave v0.1.3 up on Foundry for as long as Foundry v11 is supported, and support for Foundry v12+ will be in module versions v0.2 and up. 

I've already updated the manifest in the v0.2.0-alpha pre-release builds to be Foundry v12 only, so there's nothing more to do.
